script/utils/check-task-numbering.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Can we put this in `src/improvements/script`? I don't think we should be applying this to old tasks.

Reply:
Also this PR is behind I think and needs updated.

Reply:
Done